Commands/Public/wolfram.js

```diff
@@ -0,0 +1,50 @@
+const fetch = require("chainfetch");
+const { tokens: { wolframAppID } } = require("../../Configurations/auth");
+
+module.exports = async ({ Constants: { Colors, APIs, UserAgent, Text } }, documents, msg, commandData) => {
+	if (!msg.suffix) {
+		return msg.send({
+			embed: {
+				color: Colors.INVALID,
+				description: Text.INVALID_USAGE(commandData, msg.guild.commandPrefix),
+			},
+		});
+	}
+
+	const response = await msg.send(Text.THIRD_PARTY_FETCH("Fetching data from Wolfram|Alpha"));
+
+	const query = encodeURIComponent(msg.suffix);
+	const body = (await fetch.get(APIs.WOLFRAM(wolframAppID, query)).set("User-Agent", UserAgent).onlyBody()).queryresult;
+	if (body.success && body.pods.length) {
+		const fields = body.pods.map(pod => ({
+			name: pod.title,
+			value: pod.subpods[0].plaintext && pod.subpods[0].plaintext !== "" ? pod.subpods[0].plaintext : `[Image](${pod.subpods[0].img.src})`,
```

Comment:
Should use the image prop

Reply:
One wolfram query could return multiple images

Reply:
You're only using the first image anyways, so what gives?

Reply:
We're using the image of the first subpod of every pod. Every pod still gets their own field, and each pod can have an image.